blocks/callout/callout.css

```diff
@@ -1,6 +1,6 @@
 .section .callout-wrapper {
   padding: 0;
-  margin-bottom: 50px;
+  margin: 50px 0;
```

Comment:
I believe this is only required for mobile/tab layout. For desktop, we should use `margin: auto`

Reply:
You are right! Missed that!

Reply:
Oh you're right, I did not realize it was aligning to the left!

What I wanted to do here is to have margins top and bottom set to 50px, and auto on the sides.